components/github/README.md

```diff
@@ -1,257 +1,12 @@
-      text: "Hello, world!",
-      channel: "#general",
-    })
-  }
-});
-```
-
-### Destinations
-
-[Destinations](https://pipedream.com/docs/destinations/), like actions, abstract the connection, batching, and delivery logic required to send events to services like Amazon S3, or targets like HTTP and email.
-
-For example, sending data to an Amazon S3 bucket is as simple as calling `$send.s3()`:
-
-```javascript
-$send.s3({
-  bucket: "your-bucket-here",
-  prefix: "your-prefix/",
-  payload: event.body,
-});
-```
-
-Pipedream supports the following destinations:
-
-- [Amazon S3](https://docs.pipedream.com/destinations/s3/)
-- [Snowflake](https://docs.pipedream.com/destinations/snowflake/)
-- [HTTP](https://docs.pipedream.com/destinations/http/)
-- [Email](https://docs.pipedream.com/destinations/email/)
-- [SSE](https://docs.pipedream.com/destinations/sse/)
-
-## Contributors
-
-Thank you to everyone who has contributed to the Pipedream codebase. We appreciate you!
-
-<a href="https://github.com/PipedreamHQ/pipedream/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=PipedreamHQ/pipedream" />
-</a>
-
-## Pricing
-
-Pipedream has a [generous free tier](https://pipedream.com/docs/pricing/#developer-tier). You can run sources and workflows for free within the limits of the free tier. If you hit these limits, you can upgrade to one of our [paid tiers](https://pipedream.com/docs/pricing/).
-
-## Limits
-
-The Pipedream platform imposes some runtime limits on sources and workflows. [Read more about those in our docs](https://pipedream.com/docs/limits/).
-
-## Found a Bug? Have a Feature to suggest?
-
-Before adding an issue, please search the [existing issues](https://github.com/PipedreamHQ/pipedream/issues) or [reach out to our team](https://pipedream.com/support/) to see if a similar request already exists.
-
-If an issue exists, please [add a reaction](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-conversations-on-github) or add a comment detailing your specific use case.
-
-If an issue _doesn't_ yet exist and you need to create one, please [use the issue templates](https://github.com/PipedreamHQ/pipedream/issues/new/choose).
-
-## Security
-
-You can read about our platform security and privacy [here](https://pipedream.com/docs/privacy-and-security/).
-
-If you'd like to report a suspected vulnerability or security issue, or have any questions about the security of the product, please contact our security team at **security@pipedream.com**.
-
-## Troubleshooting
-
-Note: Event Source [New Card in Column](https://github.com/PipedreamHQ/pipedream/blob/master/components/github/sources/new-card-in-column/new-card-in-column.mjs) only supports legacy (classic) projects.
-
-Please [reach out](https://pipedream.com/support/) to the Pipedream team with any technical issues or questions about the Github integration. We're happy to help!
+**Example: New or Updated Issue (Non-Admin)**
```

Comment:
The `(Admin)` vs `(Non-Admin)` terminology is a bit confusing to me, as an uninformed user, especially since we're talking about repo-specific access, and not user-specific.

Reply:
Makes sense - dropped this.